tests/e2e/network/node.go

```diff
@@ -197,7 +197,7 @@ var _ = Describe("Azure node resources", Label(utils.TestSuiteLabelNode), func()
 
 		utils.Logf("getting all NICs of VMSS VMs")
 		for _, vmssVM := range vmssVMs {
-			utils.Logf("Checking %d VMSS VM %q", vmssVM.Name)
+			utils.Logf("Checking VMSS VM %q", *vmssVM.Name)
```

Comment:
shall we safely refer to this variable securely by invoking to. String?

Reply:
Updated.